compass/machines/badger.cfg

```diff
@@ -37,6 +37,13 @@ spack = /usr/projects/climate/SHARED_CLIMATE/compass/badger/spack
 use_e3sm_hdf5_netcdf = False
 
 
+# The parallel section describes options related to running jobs in parallel
+[parallel]
+
+# account for running diagnostics jobs
+account = t22_ocean_time_step
```

Comment:
It makes sense to have the default be the account that the most people have access to.  Trevor, Holly, and I won't be able to use this for MALI tests (unless Mark wanted to add us, but I wouldn't expect that to happen just for this reason).  It would be convenient if for us we could default to `t22_iceocean`, but I expect making a core-specific default is not worth the trouble.  A different option is that if the account is *not* included at all in the job script, SLURM will use the user's default account (which for me is set to `t22_iceocean`).  The only downside there is if a new user has not set a default account, but I would suggest it makes sense for users to do that on LANL IC.  So I guess my preference would be for account to not be included in the Badger job script.

  @xylar , @mark-petersen , what do you think about this issue?

Reply:
I agree, no default account is best.  I have taken it out (left it blank).

Reply:
> It would be convenient if for us we could default to `t22_iceocean`, but I expect making a core-specific default is not worth the trouble.

This would be possible but hard to maintain.  There isn't currently a good way to have config options for an MPAS core and machine.  We could add `account_ocean` and `account_landice` config options that get used if `account` is blank.  But the `t22_*` accounts are obviously a moving target whereas the default account seems easy.  If folks haven't set a default, they will just have to weight the relative pain of providing a config file or setting a default.